netlify/edge-functions/repo-page-card.tsx

```diff
@@ -179,7 +179,7 @@ export default async function handler(req: Request) {
                 <span style={statValueContainerStyles}>
                   <span style={statValueStyles}>{humanizeNumber(forkCount)}</span>
                 </span>
-                <span style={statSubTextStyles}>Stars</span>
+                <span style={statSubTextStyles}>Forks</span>
```

Comment:
oof. how'd I miss that. 🙈

Reply:
😅  I was about to share it and noticed it

Reply:
😅